staff/marti-malek/gamma-template/style/style.sass

```diff
@@ -0,0 +1,701 @@
+    p
+        color: white
+        padding: 1rem
+        text-align: center
+    img
+        align-self: center
+        width: 60%
+        padding: 1rem
+
+.same
+    @extend %section
+
+.last
+    background-color: #1baa98
+    width: 100%
+    margin-top: 1rem
+    h1
+        color: white
+        padding: 1rem
+    ul
+        list-style: none
+        color: white
+    ul li:before
+        content: '✓ '
+        line-height: 30px
+    img
+        width: 50%
+        align-self: center
+        padding: 2rem
+        padding-top: 1rem
+        margin: auto   
+
+.foot
+    background-color: #1b222a
+    text-align: center
+    width: 100%
+    nav
+        padding-top: 1rem
+        a
+            text-decoration: none
+            font-size: 0.8rem
+            color: white
+            padding: 1rem
+    p
+        color: white
+        padding: 1rem
+        text-align: center
+
+@media (min-width: 580px)
+    .first
+        display: inline-flex
+        background: #1c222a url(site_showcase_bg.jpg) no-repeat top center
+        background-size: 100% 100%
+        h1
+            margin: 0rem 5rem
+        p
+            margin: 0rem 5rem
+        img
+            width: 30%
+    .same
+        img
+            width: 90%
+    .last
+        h1
+            font-size: 2rem
+            padding-inline-start: 40px
+        ul
+            font-size: 1.5rem
+    .foot
+        display: grid
+        grid-template-columns: 1fr 1fr
+        grid-template-rows: 1fr 1fr
+        nav
+            grid-column: 1 / 2
+            grid-row: 1 / 2
+            a
+                padding: 0.5rem
+        p
+            grid-column: 2 / 3
+            grid-row: 2 / 3
+
+@media (min-width: 900px)
+    .home-h
+        div
+            nav
+                display: flex
+                align-items: center
+                :nth-child(1)
+                    background-color: lightgray
+                a
+                    text-decoration: none
+                    color: black
+                    font-size: 1rem
+                    padding: 1rem
+        button
+            display: none
+    .first
+        h1
+            margin-right: 30rem
+            font-size: 2.7rem
+        p
+            margin-right: 17rem
+        :nth-child(4)
+            margin-right: 8rem
+    .last
+        img
+            width: 40%
+@media (min-width: 1400px)
+    .first
+        display: grid
+        margin-bottom: 0
+        grid-template-columns: 1fr 1fr
+        grid-template-rows: 50px 100px 100px 100px 100px
+        h1
+            grid-column: 1 / 2
+            grid-row: 2 / 3
+            margin: 0
+            font-size: 4rem
+            margin-left: 4rem
+            justify-self: left
+        p
+            grid-column: 1 / 2
+            grid-row: 3 / 4
+            margin: 0
+            margin-left: 4rem
+            font-size: 1.8rem
+            padding-right: 2rem
+            text-align: left
+        :nth-child(3)
```

Comment:
Be carefull with the nth-child() in scalable projects

Reply:
Should I change something?

Reply:
No, but keep it in mind 🤓